packages/web-components/src/AgoricWalletConnection.js

```diff
@@ -122,13 +122,15 @@ export const makeAgoricWalletConnection = (makeCapTP = defaultMakeCapTP) =>
           },
         });
         this.dispatchEvent(ev);
-        this.requestUpdate();
+        if (requestUpdate) {
+          this.requestUpdate();
```

Comment:
Looks like removing this altogether breaks things, but it works fine removing it from `firstUpdated`.

Reply:
Great!  Thanks for fixing this.